filebeat/harvester/reader/log.go

```diff
@@ -113,16 +116,19 @@ func (r *logFileReader) Read(buf []byte) (int, error) {
 			return n, ErrInactive
 		}
 
-		if r.config.ForceClose {
-			// Check if the file name exists (see #93)
+		if r.config.CloseRenamed {
+			if !file.IsSameFile(r.fs.Name(), info) {
```

Comment:
I added `Check if the file has been renamed`. It does not check if the file still exists, it only checks if the file with the same name is this file or an other file. In case no file under the same name exists, it just assumes rename (it could also be removed). But that is checked in the next check.